assets/js/googlesitekit/datastore/site/info.js

```diff
@@ -905,6 +909,16 @@ export const selectors = {
 	 * @return {boolean|undefined} `true` if registrations are open; `false` if not. Returns `undefined` if not yet loaded.
 	 */
 	getAnyoneCanRegister: getSiteInfoProperty( 'anyoneCanRegister' ),
+
+	/**
+	 * Checks if WordPress site is running in the multisite mode.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {boolean|undefined} `true` if it is multisite; `false` if not. Returns `undefined` if not yet loaded.
+	 */
+	getIsMultisite: getSiteInfoProperty( 'isMultisite' ),
```

Comment:
As a selector for boolean, this can simply be `isMultisite` (similar to `isGatheringData`, `isAMP`, etc).